### PR TITLE
Cli: export-recovery-device

### LINKED ISSUE #6070: CLI: `export-recovery-device`

### PR DESCRIPTION
**Pull request by TimeEngineer**
closes #6070

### CODE REVIEW

File: cli/src/tests.rs [hunk 1]
```diff
@@ -267,6 +267,33 @@ async fn cancel_invitation(tmp_path: TmpPath) {
         ));
 }
 
+#[rstest::rstest]
+#[tokio::test]
+async fn export_recovery_device(tmp_path: TmpPath) {
```
Comment (FirelightFlagboy) [inline cli/src/tests.rs:272]: Is it possible to check if `output` is populated with the expected data ?

Comment (TimeEngineer) [reply cli/src/tests.rs:272]: Sure

File: cli/src/tests.rs [hunk 2]
```diff
@@ -267,6 +267,35 @@ async fn cancel_invitation(tmp_path: TmpPath) {
...
+            &alice.slughash(),
+            "--output",
+            &output.to_string_lossy(),
+        ])
+        .assert()
+        .stdout(predicates::str::contains(
+            "Saving recovery device file\nSaved in",
+        ));
+
+    assert!(output.exists());
```
Comment (FirelightFlagboy) [inline cli/src/tests.rs:296]: Nice, but is it possible to check the content ? or that would be redundant with `save_recovery_device()` tests ?

Comment (TimeEngineer) [reply cli/src/tests.rs:296]: The `cli` is not able to know what's inside this file, and yes it is a low level test (binary file)

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**